crates/bevy_input/src/button_input.rs

```diff
@@ -29,6 +29,15 @@ use bevy_ecs::schedule::State;
 /// * Using [`ButtonInput::clear_just_pressed`] or [`ButtonInput::clear_just_released`] instead.
 /// * Calling [`ButtonInput::clear`] or [`ButtonInput::reset`] immediately after the state change.
 ///
+/// ## Window focus
+///
+/// `ButtonInput<KeyCode>` is tied to window focus. For example, if the user holds a button
+/// while the window looses focus, [`ButtonInput::just_released`] will trigger; if the window
+/// regains focus, [`ButtonInput::just_pressed`] will trigger. Currently this happens even if the
+/// focus switches from one Bevy window to another (for example because a new window was just spawned).
```

Comment:
```suggestion
/// `ButtonInput<KeyCode>` is tied to window focus. For example, if the user holds a button
/// while the window loses focus, [`ButtonInput::just_released`] will be triggered. Similarly if the window
/// regains focus, [`ButtonInput::just_pressed`] will be triggered. Currently this happens even if the
/// focus switches from one Bevy window to another (for example because a new window was just spawned).
```